kahuna/public/js/components/gr-title/gr-title.js

```diff
@@ -0,0 +1,20 @@
+import angular from 'angular';
+import 'titip';
+
+export const module = angular.module('grTitle', []);
+
+module.directive('grTitle', [function () {
+    return {
+        restrict: 'A',
+        link: function (scope, element, attrs) {
+            const theme = attrs.grTitleTheme || 'default';
+            const position = attrs.grTitlePosition || 'bottom';
+
+            element.attr('data-title', attrs.grTitle)
+                .addClass(`titip-${theme}`)
+                .addClass(`titip-${position}`);
+
+            element.removeAttr('title');
```

Comment:
In this case, it would be better to encourage/enforce not having both this directive and a title on the same element? Could noisily fail (throw or log error) if that were the case so we can fix the template?